### PR TITLE
Translate new `rmpinfo` functions

### PR DESCRIPTION
**Pull request by Fan2Shrek**

### CODE REVIEW

File: reference/rpminfo/functions/rpmdefine.xml [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+  &reftitle.description;
+  <methodsynopsis>
+   <type>bool</type><methodname>rpmdefine</methodname>
+   <methodparam><type>string</type><parameter>text</parameter></methodparam>
+  </methodsynopsis>
+  <para>
+   Définit ou change la valeur d'une macro RPM.
+  </para>
+  <para>
+   Cela peut être utilisé pour sélectionner le chemin de la base de données et le "backend" à utiliser
```
Comment (Fan2Shrek) [inline reference/rpminfo/functions/rpmdefine.xml:20]: No sure how to translate "backend" here

Comment (devnexen) [reply reference/rpminfo/functions/rpmdefine.xml:20]: keeping backend is fine, it s hard in this case not to find a too contrived wording in this case

**Review by devnexen — APPROVED** [review]

File: reference/rpminfo/functions/rpmdefine.xml (hunk 1 above)
Comment (remicollet) [reply reference/rpminfo/functions/rpmdefine.xml:20]: "moteur" is commonly used to describe database engine ;)

https://fr.wikipedia.org/wiki/Moteur_de_base_de_donn%C3%A9es

**Comment by remicollet** [comment]
@Fan2Shrek thanks again for translating my work
(I could have done it, but you were faster)

**Pull request merged**